### PR TITLE
various fixes from related Trello ticket for 1.2.0 release

### PR DESCRIPTION
**Pull request by dojosout**
Fixed issues from Trello needed for 1.2.0 release

### CODE REVIEW

File: dashboards/flow-data-per-country.json [hunk 1]
```diff
@@ -399,12 +399,12 @@
       "type": "text"
     },
     {
-      "content": "<h1><center>Top Sources to $Country</center></h1>",
+      "content": "<h1><center>Top Flows Received by $Country</center></h1>",
```
Comment (daldoyle) [inline dashboards/flow-data-per-country.json]: This feels to me like it's going back into the discussion of sender/receiver vs source/destination that we've gone back and forth on a lot of times. I believe the final decision was to use source/destination if at all possible. Do we think "Top Sources to Country" is too ugly? Maybe we should call this "Top Talkers to $Country"? Something else?

File: dashboards/flow-data-per-country.json [hunk 2]
```diff
@@ -1017,12 +1017,12 @@
       }
     },
     {
-      "content": "<h1 style=\"margin-top:35px;\"><center>Top Destinations from $Country</center></h1>",
+      "content": "<h1 style=\"margin-top:35px;\"><center>Top Flows Sent by $Country</center></h1>",
```
Comment (daldoyle) [inline dashboards/flow-data-per-country.json]: See above comment.

**Comment by daldoyle** [comment]
I think in light of all the header cleanups we should probably close this PR out as "overtaken by events"? ie I don't think anything in here is applicable anymore. @dojosout can you please comment and close if you agree?

**Comment by dojosout** [comment]
Agreed. Closing.

**Pull request closed without merging**